src/main/kotlin/no/nav/amt/deltaker/bff/endringsmelding/Endringsmelding.kt

```diff
@@ -0,0 +1,97 @@
+package no.nav.amt.deltaker.bff.endringsmelding
+
+import com.fasterxml.jackson.annotation.JsonSubTypes
+import com.fasterxml.jackson.annotation.JsonTypeInfo
+import java.time.LocalDate
+import java.time.LocalDateTime
+import java.util.UUID
+
+data class Endringsmelding(
+    val id: UUID,
+    val deltakerId: UUID,
+    val utfortAvNavAnsattId: UUID?,
```

Comment:
Her fraviker det fra slik det er gjort ellers i appen, hvor vi bruker navident isteden. Jeg vet ikke helt hva som er lurt her, på de eksisterende endringsmeldingene vil det jo være nav-ansatte som, jeg tror, ikke har noen direkte tilknyttning til denne applikasjonen så jeg er ikke helt sikker på om de skal opprettes eller ikke når vi lagrer endringsmeldingene, eventuelt hente ut bare navident fra person-service.

Reply:
Trenger vi egentlig å lese inn/vise de utførte endringsmeldingene i det hele tatt? Eller må vi det for å vise historikk på deltakere som er opprettet i Arena og som har blitt administrert på gamlemåten? 

Hvis vi skal lese inn og vise endringsmeldinger der vi ikke har nav-ansatt som har den tilhørende id-en kan vi kanskje hente dem fra amt-personservice som du sier.

Reply:
Det er nok uavklart om vi trenger å vise de eller ikke, det blir jo dårlig historikk på de som er opprettet utenfor løsningen uansett. Men det er vel bare å hente de navansatte uansett som du sier nedenfor.